test/metabase/lib/aggregation_test.cljc

```diff
@@ -245,156 +245,160 @@
     (is (= :type/Integer
            (lib/type-of query (first (lib/aggregations-metadata query)))))))
 
+(def ^:private op-test-summable-cols
```

Comment:
these column definitions were duplicated between two tests that were already super huge (the first was like 180 lines) so I split them out into top-level defs